torch/testing/_internal/common_quantization.py

```diff
@@ -785,19 +785,25 @@ def forward(self, x, hid):
         return x, hid
 
 class ConvModel(torch.nn.Module):
-    def __init__(self):
+    def __init__(self, transposed=False):
```

Comment:
can we create a new class instead, to follow that `torch.nn.Conv` is a different class than `torch.nn.ConvTranspose`?